### ISSUE TITLE
provide global caption and local caption

### ISSUE THREAD

**Issue by joyli-x** [OSU-NLP-Group/MagicBrush#23]
Hello, I didn't find global caption and local caption in the huggingface dataset. Can you provide this part of data? Thanks!

**Comment by drogozhang** [comment]
for eval or test. They are provided in [val set](https://buckeyemailosu-my.sharepoint.com/personal/zhang_13253_buckeyemail_osu_edu/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease%2Fval%2Ezip&parent=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease&ga=1) and [test set](https://buckeyemailosu-my.sharepoint.com/personal/zhang_13253_buckeyemail_osu_edu/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease%2Ftest%2Ezip&parent=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease&ga=1).

For training set, we didn't generate these files but you can generate these with the prompts provided in the paper.

**Comment by joyli-x** [comment]
> for eval or test. They are provided in [val set](https://buckeyemailosu-my.sharepoint.com/personal/zhang_13253_buckeyemail_osu_edu/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease%2Fval%2Ezip&parent=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease&ga=1) and [test set](https://buckeyemailosu-my.sharepoint.com/personal/zhang_13253_buckeyemail_osu_edu/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease%2Ftest%2Ezip&parent=%2Fpersonal%2Fzhang%5F13253%5Fbuckeyemail%5Fosu%5Fedu%2FDocuments%2FDatasets%2FMagicBrush%2FfinalRelease&ga=1).
> 
> For training set, we didn't generate these files but you can generate these with the prompts provided in the paper.

Ok, thanks!
